tests/framework/BaseYiiTest.php

```diff
@@ -99,8 +100,7 @@ public function testSetupLogger()
         $this->assertSame($logger, BaseYii::getLogger());
 
         BaseYii::setLogger(null);
-        $defaultLogger = BaseYii::getLogger();
-        $this->assertTrue($defaultLogger instanceof Logger);
+        BaseYii::getLogger();
```

Comment:
`\yii\base\InvalidConfigException` may be thrown on `Logger` initialization as well. This test should be split